addons/l10n_fi/data/account_tax_report_line.xml

```diff
@@ -15,16 +15,17 @@
         <field name="line_ids">
             <record id="tax_report_sales_title" model="account.report.line">
                 <field name="name">Tax on domestic sales by tax rates</field>
-                <field name="aggregation_formula">sale_25_5.balance + sale_24.balance + sale_14.balance + sale_10.balance</field>
+                <!-- "sale_25_5" refers to both 24% and 25.5% taxes. Will only be 25.5 on 01/01/2025.-->
+                <field name="aggregation_formula">sale_25_5.balance + sale_14.balance + sale_10.balance</field>
                 <field name="children_ids">
                     <record id="tax_report_sales_25_5" model="account.report.line">
-                        <field name="name">25.5 %:n vero</field>
+                        <field name="name">25.5 %:n vero (+ 24 %:n vero)</field>
                         <field name="code">sale_25_5</field>
                         <field name="expression_ids">
                             <record id="tax_report_sales_25_5_tag" model="account.report.expression">
                                 <field name="label">balance</field>
                                 <field name="engine">tax_tags</field>
-                                <field name="formula">fi_320</field>
+                                <field name="formula">fi_320 + fi_301</field>
```

Comment:
That's not the way to do, it will create a tag `fi_320 + fi_301` and so your tax can't find the tag `fi_320`.
You will want an expression with balance of type aggregation that will aggregate another new expression with the tax_tag, I suppose)

Reply:
My bad. I should have checked

Reply:
I guess all of the fw-ports will need to be reverted